source/MaterialXView/Glsl/external/tinyexr/TinyEXRImageHandler.h

```diff
@@ -14,6 +14,9 @@ using TinyEXRImageHandlerPtr = std::shared_ptr<class TinyEXRImageHandler>;
 class TinyEXRImageHandler : public ImageHandler
```

Comment:
Would it make sense to move this class out from external/tinyexr, and rename it to ExrImageHandler? Since the use of tinyexr could change in the future, but using the same interface for this class. And to avoid mixing local classes into the external stuff.

Reply:
I think I'd like to leave it as is for now. Even with an ExrImageHandler the TinyEXR dependency needs to reside somewhere. I will probably move this a bit higher though not sure where yet. Maybe an Imagehandler directory.